nautobot/core/jobs/__init__.py

```diff
@@ -410,14 +411,14 @@ class RunRegisteredDataComplianceRules(Job):
     description = "Runs selected Data Compliance rule classes."
 
     selected_data_compliance_rules = MultiChoiceVar(
-        choices=get_choices,
+        choices=get_data_compliance_choices,
         label="Select Data Compliance Rules",
         required=False,
         description="Not selecting any rules will run all rules listed.",
     )
 
-    run_builtin_rules_in_report = BooleanVar(
-        label="Run built-in validation rules?", description="Include created, built-in data validation rules in report."
+    run_existing_rules_in_report = BooleanVar(
+        label="Run existing validation rules?", description="Include existing data validation rules in report."
```

Comment:
"Existing" seems at least as confusing to me - we wouldn't ever run **non**-existing rules...? Might need to read the code to understand what this is actually affecting about the operation of the job.

Reply:
It seems like enabling this option will validate relevant objects against all the user created validation rules, which are `RegularExpressionValidationRule`, `MinMaxValidationRule`, `RequiredValidationRule`, and `UniqueValidationRule`.
I was confused because the data compliance objects were not being generated even tho I had those rules and non-compliant objects in place. So I thought it was something else. However, that is due to what I mentioned here https://networktocode-llc.slack.com/archives/C01RMAGBD29/p1743106959423749

Reply:
TLDR, `user-created` seems to be the most explicit and appropriate here